app/services/concerns/childcare_eligibility.rb

```diff
@@ -1,13 +1,16 @@
 module ChildcareEligibility
 private
 
-  def eligible_for_childcare_costs?(person, submission_date)
-    person_has_dependant_child?(person, submission_date) &&
+  # A person may count their childcare costs towards their assessed outgoings
+  # if either the applicant or their partner has a child dependant and if
+  # the person who has the childcare costs is either employed or a student
```

Comment:
not sure this is quite right. I think it doesn't matter who pays the costs ('it all comes out of the same bucket') but it does matter that the dependants need to belong to the student/employed person

Reply:
As per out discussion with Mike Hooper the rules are: It doesn't matter which person incurs the childcare costs or which person the dependant belongs to. What matters is that both client and partner (if there is one) are either students or employed. Calculations now updated to incorporate that.